osde2e/custom_domains_operator_tests.go

```diff
@@ -117,6 +118,11 @@ var _ = ginkgo.Describe("Custom Domains Operator", ginkgo.Ordered, func() {
 			return false
 		}).WithTimeout(5*time.Minute).WithPolling(pollInterval).Should(BeTrue(), "Endpoint never became ready")
 	})
+	ginkgo.It("upgrades operator", func(ctx context.Context) {
+		ginkgo.By("testing operator upgrade")
```

Comment:
```suggestion
	ginkgo.It("can be upgraded", func(ctx context.Context) {
```

to be consistent and the `By` isn't providing much for us (we know we are already in the upgrade operator test)

`custom-domains-operator upgrades operator` vs. `custom-domains-operator can be upgraded`

there is a section towards the bottom on test naming with ginkgo https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/writing-good-e2e-tests.md